src/rgw/setup/rgw_service.py

```diff
@@ -27,8 +28,9 @@ class RgwStart:
     def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
-        except OSError as e:
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
+            cmd = shlex.split(cmd)
```

Comment:
Also flush file descriptor always before replacing current procuss

https://docs.python.org/3/library/os.html
> The current process is replaced immediately. Open file objects and descriptors are not flushed, so if there may be data buffered on these open files, you should flush them using sys.stdout.flush() or [os.fsync()](https://docs.python.org/3/library/os.html#os.fsync) before calling an [exec*](https://docs.python.org/3/library/os.html#os.execl) function.

Reply:
At this point of execution I dont see any reference of any fd.
Added sys.stdout.flush() to send any remaining output to terminal.

Reply:
flush stderr also in similar fashion explicitly

Reply:
Done.